sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -1183,6 +1186,8 @@ pi_result cuda_piDeviceGetInfo(pi_device device, pi_device_info param_name,
 
   assert(device != nullptr);
 
+  ScopedContext active(device->get_context());
```

Comment:
Is this related to the other changes or just a small change included.

Reply:
A small change included, it's related to the issues I had in  https://github.com/intel/llvm/pull/8124 with this patch having the scoped context for the whole of `get_info` is no longer expensive.